packages/components/Input/src/Input.tsx

```diff
@@ -22,16 +18,7 @@ import { useInput } from './useInput';
  * @param userProps The props that were passed into the input
  * @returns Whether the styles that are assigned to the layer should be applied to the input
  */
-export const inputLookup = (layer: string, state: FocusState & { text: string }, userProps: InputProps): boolean => {
-  return (
-    (layer === 'focused' && state.focused && !state.text) ||
-    (layer === 'filled' && !state.focused && !!state.text) ||
-    (layer === 'typing' && state.focused && !!state.text) ||
-    userProps[layer] ||
-    (layer === 'hasIcon' && userProps.defaultIcon) ||
-    (layer === 'error' && !!userProps.error && !!state.text)
-  );
-};
+export const inputLookup = () => null;
```

Comment:
?

Reply:
Disabling for other platforms similar to Chip